test/unit/test_lambda.py

```diff
@@ -0,0 +1,293 @@
+import pytest
+import json
+
+# make sure we can find the app code
+import sys, os
+my_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, my_path + '/../../code/')
+
+import lambda_handlers as handlers
+from jsonschema.exceptions import ValidationError
+
+
+def test_fv_handler():
+    response = handlers.fv_handler(json.load(open('../fv.json')), None)
+    assert 'result' in response
+    assert round(response.get('result'), 6) == 15692.928894
```

Comment:
Is calling `response.get('result')` considered better python style than `response['result']`?

Reply:
I believe it is because it gives the option of providing a default value as a second parameterand doesn't throw an error if the key doesn't exist... though I use neither of those here.